src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+const toNumbers = population.map((element) => {
+  return parseInt(element.textContent.replace(/,/g, ''));
+});
```

Comment:
It's a bit confusing to use the variable name 'population' for an array of DOM elements, and then 'toNumbers' for the actual population numbers. Consider renaming 'population' to something like 'populationElements' or 'populationNodes', and 'toNumbers' to 'populations' or 'populationNumbers'.